cel/cos_tlv.go

```diff
@@ -79,3 +85,38 @@ func (t TLV) ParseToCosTlv() (CosTlv, error) {
 func (t TLV) IsCosTlv() bool {
 	return t.Type == CosEventType
 }
+
+// ValidateEnvVar takes in a environment name and its value, run some checks and return an error
+// if they are invalid.
+func ValidateEnvVar(name string, value string) error {
```

Comment:
Should we have this be called something like `FormatEnvVar` that also does the joining with `=`?

That way `FormatEnvVar` and `ParseEnvVar` would be inverses of eachother and any caller of `ValidateEnvVar` will always be formatting them as the very next call.

Reply:
yes that makes sense